docker-compose-tests.yml

```diff
@@ -36,7 +36,7 @@ services:
       - gp-finder-db:mongo
 
   gp-finder-db:
-    image: "nhsuk/profiles-db:${profiles_db_docker_image_tag-0.3.0}"
+    image: "nhsuk/profiles-db:${profiles_db_docker_image_tag-0.4.0}"
```

Comment:
Is there a need to use a specific database? Wouldn't that be something you change ad-hoc? Just wondering...